### PR TITLE
Misc queue from 2018 (sorry, lost sources) 

### PR DESCRIPTION
**Pull request by Gelma**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -2208,8 +2215,11 @@ capela->capella
 caperbility->capability
 Capetown->Cape Town
 capible->capable
+capitalise->capitalize
 captable->capable
+capter->captor
```
Comment (arm-in) [inline codespell_lib/data/dictionary.txt]: also "chapter"

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -2208,8 +2215,11 @@ capela->capella
 caperbility->capability
 Capetown->Cape Town
 capible->capable
+capitalise->capitalize
```
Comment (arm-in) [inline codespell_lib/data/dictionary.txt]: British English OK

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: thanks. done.

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: ok. removed.

**Pull request closed without merging**